backend/tests/core/test_async_loop_issues.py

```
@@ -0,0 +1,320 @@
+            mock_session.close.assert_called_once()
+
+    @pytest.mark.asyncio
+    async def test_session_with_custom_timeout(self) -> None:
+        """Test that custom timeout is passed to session."""
+        with patch("aiohttp.ClientSession") as mock_session_class:
+            with patch("aiohttp.ClientTimeout") as mock_timeout_class:
+                mock_session = AsyncMock()
+                mock_session.close = AsyncMock()
+                mock_session_class.return_value = mock_session
+
+                async with AsyncSessionManager(timeout=60.0) as session:
```

Ruff (0.14.14)

[error] Local variable `session` is assigned to but never used

Remove assignment to unused variable `session`

(F841)